content/id/docs/concepts/storage/dynamic-provisioning.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Penyediaan Volume Dinamis
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+Penyediaan volume dinamis memungkinkan volume penyimpanan untuk dibuat sesuai permintaan (_on-demand_).
+Tanpa adanya penyediaan dinamis (_dynamic provisioning_), untuk membuat volume penyimpanan baru, admin kluster haruslah secara manual
+memanggil penyedia layanan cloud atau layanan penyimpanan, dan kemudian membuat [objek PersistentVolume](/docs/concepts/storage/persistent-volumes/)
+sebagai representasi di Kubernetes. Fitur penyediaan dinamis menghilangkan kebutuhan admin kluster untuk menyediakan
+penyimpanan sebelumnya (_pre-provision_). Namun, penyimpanan akan tersedia secara otomatis
```

Comment:
Namun -> Dengan demikian

Reply:
addressed in 490d25dd8a13b115293327a3e7ad9009052f2212